se/se_epub_lint.py

```diff
@@ -1153,9 +1153,12 @@ def lint(self, skip_lint_ignore: bool) -> list:
 									if glossary_value[1] is False and glossary_value[0] in text:
 										glossary_usage[glossary_index] = (glossary_value[0], True)
 					# Finally, log any entries that don’t exist in the text
+					entries = []
 					for glossary_value in glossary_usage:
 						if glossary_value[1] is False:
-							messages.append(LintMessage("m-070", f"Glossary entry [text]{glossary_value[0]}[/] not present in the text.", se.MESSAGE_TYPE_ERROR, filename))
+							entries.append(glossary_value[0])
+					entries_text = ", ".join(entries)
+					messages.append(LintMessage("m-070", f"Glossary entries not present in the text: [text]{entries_text}[/]", se.MESSAGE_TYPE_ERROR, filename))
```

Comment:
That's not quite what I meant. What I meant is, lint errors can have any number of submessages (i.e. in the output table, the next line is dimmed and has a right arrow pointing to it. The main error should be the generic message `Glossary entries not present in the text.` Then each detected problem entry should be a submessage. You just pass a list of strings as the last option to the LintMessage constructor.

Reply:
Aha, that was new to me. Fixed in https://github.com/standardebooks/tools/pull/440/commits/e1007e2ddf6d4150fb3df3da2278c1a28163b53a